tools/scripts/windows/run-client.ps1

```diff
@@ -0,0 +1,30 @@
+. $PSScriptRoot\env.ps1
+
+$BUFFER_SIZE="1024"
+$NCLIENTS="8"
+
+if ($args.Count -ne 1) {
+    Write-Host "Please input the server address."
+    Write-Host "Usage: run-cli.ps1 {server IP address}"
+    exit
+}
+
+$SERVER_HOST = $args[0]
+
+Write-Host " *******        Client Mode         ********************* "
+Write-Host " LIBOS:           $env:LIBOS"
+Write-Host " CONFIG:          $env:CONFIG_PATH"
```

Comment:
Since this requires setting the `$env:LIBOS` before running this script, it may lead to unknowns to the user. It would be good to add complete, opinionated examples using these scripts which will lead to total clarity as to how to use them.

Otherwise, it's equivalent to using the Make system and running programs with appropriate arguments.

Reply:
We already have steps explaining how to setup and build checked into the `doc` directory. Are those not enough to run these tests? If they are ambitious or incomplete, then we should modify those scripts instead of creating new documents because they will go out of sync soon. Look at the `.md` files in the `doc` dir. (e.g. https://github.com/microsoft/demikernel/blob/dev/doc/setup.md)